docs/video/obs/docks.md

```diff
@@ -20,19 +20,22 @@ Displays useful statistics regarding your recording and/or streaming status, mos
 * Skipped/missed frames due to encoding/rendering lag
 * Disk space available / Approximation of how long it'll take for the recording to fill up the disk
 
-!!! important "There are ways to open stat docks"
+!!! important "There 2 are ways to view the stats dock"
 
-    1. There is one available in <kbd>Docks</kbd> that you can dock into your interface (like previously shown)
-    2. There is another in <kbd>View</kbd> that you can enable in <kbd>View</kbd> which is not dockable in the window and stays a separate floating window
+    1. Enabling it in <kbd>Docks</kbd> lets you dock it into your interface (like previously shown)
+    2. Enabling it in <kbd>View</kbd> stays a separate floating window
 
 
 
 ## :material-projector-screen-outline: Scenes
 
-Scenes are collections of user-defined sources, to give you an idea they’re typically used by streamers who switch from game source w/ facecam, full-screen facecam,and some « starting stream soon / brb » text 
+Scenes are collections of user-defined sources, to give you an idea they’re typically used by streamers who switch from game source w/ facecam, full-screen facecam, and starting stream soon / brb text.
+<!--
+Those arrow symbols just look wacky, decided to remove them.
+-->
```

Comment:
could you replace them with " instead of (what i think they're called) french quotes